pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/advisor.go

```diff
@@ -78,9 +79,8 @@ type cpuResourceAdvisor struct {
 	startTime   time.Time
 	systemNumas machine.CPUSet
 
-	regionMap          map[string]region.QoSRegion              // map[regionName]region
-	containerRegionMap map[string]map[string][]region.QoSRegion // map[podUID][containerName]regions
-	poolRegionMap      map[string][]region.QoSRegion            // map[poolName]regions
+	regionMap     map[string]region.QoSRegion   // map[regionName]region
+	poolRegionMap map[string][]region.QoSRegion // map[poolName]regions
```

Comment:
Move pool region names to pool info in metacache as region names in container info.